crates/vm/src/system/memory/online.rs

```diff
@@ -650,12 +650,13 @@ impl TracingMemory {
     }
 
     #[inline(always)]
-    fn split_by_meta<T: Copy, const MIN_BLOCK_SIZE: usize>(
+    fn split_by_meta<const MIN_BLOCK_SIZE: usize>(
         &mut self,
         start_ptr: u32,
         timestamp: u32,
         block_size: u8,
         address_space: usize,
+        cell_size: usize,
```

Comment:
isn't this worse for performance though? if you can know the size statically, that is better